.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+Please start by choosing an appropriate base branch for your pull request.
+
+Making a quick fix for a problem observed on production service? Fixing a minor
+bug, improving existing documentation or record metadata? Make your pull request
+against the ``production`` branch.
+
+Introducing a new feature? Make your pull request against the ``master`` branch.
+
+Please reference all issues that this PR addresses or closes. Do not forget to
+include ``(addresses #123)`` or ``(closes #123)`` in your commit log message.
+
+Please run the tests locally in order to speed up developments and save Travis
+CI cycles.  Look for ``./run-tests.sh``.
+
+Finally, before submitting the pull request, please select an appropriate issue
+labels and the concrete milestone to ease our housekeeping.
```

Comment:
Labels and milestones can only be assigned by team members. Remove mention?

Reply:
Used again the introductory "if you are a member..." that should address this too. Although we very rarely get pull requests from outside the organisation... I recall only a couple of examples from the past.